exercises/verify.cpp

```diff
@@ -7,12 +7,30 @@ The number 3 is [not] present in the array.
 */
 
 #include <iostream>
+#define EXIT_SUCCESS 0
 using namespace std;
 
 int main()
 {
   // placeholder
   int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
+  int number;
+  bool flag= false;
+  cout << "Insert number";
+  cin >> number;
+  for (int i = 0; i < 10; i++) {
+	  if (N[i] == number){
+		  flag = true;
+		  break;
+	  }
+  }
+  if (flag == true) {
+	  cout << "The number %d is present in the array", number;
+  }
+  else {
+	  cout << "The number %d is not present in the array", number;
+  }
```

Comment:
In questo `if` è presente del codice ridondante: potresti mettere un operatore condizionale ternario. 
Puoi dare un'occhiata [qui](https://www.w3schools.com/cpp/cpp_conditions_shorthand.asp) per provare a sostituire, per esempio: 
```cpp
cout <<  "..." << flag ? "" : " not " << "...";
```